radicle-httpd/src/api/v1/projects.rs

```diff
@@ -939,4 +999,51 @@ mod routes {
             ])
         );
     }
+
+    #[tokio::test]
+    async fn test_projects_patches() {
+        let tmp = tempfile::tempdir().unwrap();
+        let app = super::router(test::seed(tmp.path()));
+        let response = get(&app, "/projects/rad:z4FucBZHZMCsxTyQE1dfE2YR59Qbp/patches").await;
+
+        assert_eq!(response.status(), StatusCode::OK);
+        assert_eq!(
+            response.json().await,
+            json!([
+              {
+                "id": "5de9f17ca5326258412ab02f9a5339b6482198ce",
+                "title": "A new `hello word`",
+                "description": "change `hello world` in README to something else",
+                "state": "proposed",
+                "target": "delegates",
```

Comment:
notice the difference between `patch` and `issue` model:

- `issue` returns author too. should I add it for `patch`?
- `issue` state is something like:
```
"state": {
  "status": "open"
}
```
whereas `patch` is simply:
```
"state": "proposed"
```

Reply:
Right, it's because you can have `state: { status: "closed", reason: "solved" }` for issue, but not for patch. We could think of a better representation, but that's the reason at least.

Reply:
Patch has an author method you can use, yeah.

Reply:
> Right, it's because you can have `state: { status: "closed", reason: "solved" }` for issue, but not for patch. We could think of a better representation, but that's the reason at least.

Yea, I think `patch` does have similar enumeration as `issue`. PRs could be closed for some reason as well, etc.

Reply:
> Patch has an author method you can use, yeah.

[added](https://github.com/radicle-dev/heartwood/compare/73b80dc7107a92347073747c4c24715e3d2b6caf..6324139868f81933d57cf5be9d4689bf193496b3#diff-85a73839d407aa508b9e78b3756b0e9b6e8d1087a0cafac6a8325a8cf427b65eR448)

Reply:
> Yea, I think `patch` does have similar enumeration as `issue`. PRs could be closed for some reason as well, etc.

True, can you open an issue for this?

Reply:
https://github.com/radicle-dev/heartwood/issues/270